src/main/java/Duke.java

```diff
@@ -1,10 +1,198 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static final String GREETING_MESSAGE = "Hai, Ningensama-tachi! Kon-Nakiri!";
```

Comment:
I like your use of constant names, they are the correct style being all uppercase and having an underscore to separate the words